samples/nft_deployment/README.md

```diff
@@ -0,0 +1,296 @@
+  admin: # what is that used for?
+    certs:
+      - /Users/bur/Developer/gocode/src/github.com/hyperledger-labs/fabric-token-sdk/samples/nft/testdata/fsc/crypto/peerOrganizations/fsc.example.com/users/Admin@fsc.example.com/msp/signcerts/Admin@fsc.example.com-cert.pem
+  tls:
+    # this is for grpc-tls connection
+  web:
+  tracing:
+  metrics:
+  endpoint:
+    resolvers:
+    # resolvers used for p2p authentication
+    - name: issuer
+      domain: fsc.example.com
+      identity:
+        id: issuer
+        path: $FSC_CRYPTO/peerOrganizations/fsc.example.com/peers/issuer.fsc.example.com/msp/signcerts/issuer.fsc.example.com-cert.pem
+      addresses:
+        P2P: 127.0.0.1:20001
+      aliases:
+
+# this section contains the configuration related to the fabric driver 
+fabric:
+  default:
+    mspConfigPath: $CRYTPO/peerOrganizations/org2.example.com/peers/alice.org2.example.com/msp
+    msps:
+      - id: idemix
+        mspType: idemix
+        mspID: IdemixOrgMSP
+        cacheSize: 0
+        path: $CRYTPO/peerOrganizations/org2.example.com/peers/alice.org2.example.com/extraids/idemix
+    tls:
+    peers:
+    channel:
+    vault:
+    endpint:
+      resolvers:
+        # can't we use discovery here?
+
+# this section contains the configuration related to the Token SDK
+token:
+  enabled: true
+  tms:
+    - certification: null
+      channel: testchannel
+      namespace: zkat
+      network: default
+      wallets:
+        owners:
+          - default: true
+            id: alice
+            path: $TOKEN_CRYPTO/default-testchannel-zkat/idemix/alice
+  ttxdb:
+    persistence:
+      opts:
+        path: $WD/testdata/fsc/nodes/alice/kvs
+      type: badger
+```
+
+### Run
+
+```bash
+cd nodes/alice
+go build -o alice
+./alice node start
+```
+
+## Token Validation Chaincode
+
+We begin by building all components used in the nft sample.
+The Token SDK uses a Token Validation Chaincode with Fabric to translate Token Transaction into Fabric Transaction with RWSets.
+See more details in [TODO])().
+
+### Prepare public parameters
+
+We are using `tokengen` to create the public parameters used by the Token Validation Chaincode in our nft sample.
+
+tokengen gen fabtoken
+
+```bash
+go install github.com/hyperledger-labs/fabric-token-sdk/cmd/tokengen
+tokengen gen dlog \ 
+  --auditors $IDEMIX_CRYTPO/peerOrganizations/org1.example.com/peers/auditor.org1.example/com/extraid/idemix/msp \ 
+  --issuers $IDEMIX_CRYTPO/peerOrganizations/org1.example.com/peers/issuer.org1.example/com/extraid/idemix/msp\
+  --cc
+
+cat zkatdlog_pp.json | base64
+```
+
+Copy the base64-encoded params into `$FTS_PATH/token/services/network/tcc/params.go`
+
+```go
+package tcc
+
+const Params = `>>>BASE64_STRING<<<`
+```
+
+
+### Build
+
+Once you have updated the `params.go`, you can compile TCC and package it using a `Dockerfile`.
+
+```bash
+go build $FTS_PATH/token/services/network/tcc/main
+```
+
+TODO
+
+### Deploy
+
+Run it as chaincode-as-a-service
```

Comment:
why chaincode as a service is deployed here.
should the chaincode been deployed together with fabric network set up? as a part of network set up?